docs/sources/source/nlp/glue.rst

```diff
@@ -0,0 +1,168 @@
+
+All the tasks are classification tasks, except for the STS-B task which is a regression task.
+All classification tasks are 2-class tasks, except for the MNLI task which is a 3-class task.
+
+More details about GLUE benchmark could be found `here <https://gluebenchmark.com/tasks>`_.
+
+Training the model
+------------------
+Before running ``examples/nlp/glue_benchmark/glue_benchmark_with_bert.py``, download the GLUE data with `this script <https://gist.github.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e>`_ by running:
+
+.. code-block:: bash
+
+    # download the script to get the GLUE data
+    wget https://gist.githubusercontent.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e/raw/17b8dd0d724281ed7c3b2aeeda662b92809aadd5/download_glue_data.py
+    # run the script to download the GLUE data
+    python download_glue_data.py
+
+After running the above commands, you will have a folder ``glue_data`` with data folders for every GLUE task. For example, data for MRPC task would be under ``glue_data/MRPC``.
+Use ``--task_name`` argument to run the training script on a specific task, use lower cased task name: ``cola, sst-2, mrpc, sts-b, qqp, mnli, qnli, rte, wnli``.
+
+The GLUE tasks can be fine-tuned on 4 pre-trained back-bone models supported in NeMo: Megatron-LM BERT, BERT, AlBERT and RoBERTa.
+See the list of available pre-trained Huggingface models `here <https://huggingface.co/transformers/pretrained_models.html>`__. 
+To get the list of all NeMo supported pre-trained models run:
+
+.. code-block:: python
+    
+    import nemo.collections.nlp as nemo_nlp
+    nemo_nlp.nm.trainables.get_pretrained_lm_models_list()
+
+Specify the model to use for training with ``--pretrained_model_name``.
+
+.. note::
+    It's recommended to finetune the model on each task separately.
+    Also, based on `GLUE Benchmark FAQ#12 <https://gluebenchmark.com/faq>`_,
+    there are might be some differences in dev/test distributions for QQP task
+    and in train/dev for WNLI task.
+
+Model training
+--------------
+
+To run the script on MRPC task on a single GPU, run:
+    
+    .. code-block:: python
+
+        python glue_benchmark_with_bert.py  \
+            --data_dir /path_to_data_dir/MRPC \
+            --task_name mrpc \
+            --work_dir /path_to_output_folder \
+            --pretrained_model_name bert-base-uncased 
+            
+
+To use multi-gpu training on MNLI task, run:
+
+    .. code-block:: bash
+
+        export NUM_GPUS=4
+        python -m torch.distributed.launch --nproc_per_node=$NUM_GPUS glue_benchmark_with_bert.py \
+            --data_dir=/path_to_data/MNLI \
+            --task_name mnli \
+            --work_dir /path_to_output_folder \
+            --num_gpus=$NUM_GPUS \
+            --pretrained_model_name bert-base-uncased \
+
+
```

Comment:
How about giving a link how to use multi-GPU with NeMo?

Reply:
done